src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -35,32 +39,46 @@
  * When everything has been changed to private, the deprecated note can be
  * removed.
  */
+@SuppressWarnings("deprecation")
 public class DLNAMediaAudio extends DLNAMediaLang implements Cloneable {
 	private static final Logger LOGGER = LoggerFactory.getLogger(DLNAMediaAudio.class);
 	private AudioProperties audioProperties = new AudioProperties();
 
+	/** The default bit rate used if the actual value is unknown */
 	public static final int BITRATE_DEFAULT = 8000;
+
+	/** The default number of audio channels used if the actual value is unknown */
 	public static final int NUMBEROFCHANNELS_DEFAULT = 2;
+
+	/** The default bits per sample used if the actual value is unknown */
 	public static final int BITSPERSAMPLE_DEFAULT = 16;
+
+	/** The default audio delay used if the actual value is unknown */
 	public static final int AUDIODELAY_DEFAULT = 0;
-	public static final int SAMPLEFREQUENCY_DEFAULT = 48000;
+
+	/** The default sample rate used if the actual value is unknown */
+	public static final int SAMPLERATE_DEFAULT = 48000;
```

Comment:
I cannot remember where i did a remark about this value, but i remember why i did chosen it, and still agree with it.
It was because MediaInfo have some limitations and when it doesn't give any values, most of the time it is 48000 Hz.

I keeped the 44100 for the no audio file commit that i've done, and become confused about that choice. red fish's memory symptome :wink: